src/drive/locales/en.json

```diff
@@ -87,6 +90,14 @@
         "action": "Authorize access",
         "success": "The application has access to your contacts"
       }
+    },
+    "banner": {
+      "shared_from": "Shared from the cozy of",
+      "synchronise": "Synchronise this content in your cozy to be able to modify it or to keep it in case of cancellation of sharing.",
+      "whats_cozy": "Cozy is a secure digital home for all your data, hosted in France - ",
+      "know_more": "More information",
+      "sync_to_mine": "Synchronise in my Cozy",
```

Comment:
I think "Sync to my Cozy" would be better.

Reply:
783eaa6ef7705148c2eeb0b44e08aba34aeef3de